tests/test_utils.py

```diff
@@ -16,7 +16,9 @@ def __init__(self, features, lat="lat", lon="lon"):
 
     @property
     def columns(self):
-        return list(self.features[0]['properties'].keys())
+        properties = list(self.features[0]['properties'].keys())
+        properties.extend([self.lat, self.lon])
```

Comment:
@ustroetz do we need to add the `lat` and `long` columns to the properties here?

Reply:
Yeah, I was looking for the best way to replicate the issue described in #12. Since we are only mocking a df based on a GeoJSON, we need to mock that a df has lat/lon as columns.